Tests/OpenCombineTests/Helpers/TestingThreadSafety.swift

```diff
@@ -6,42 +6,63 @@
 //
 
 #if !WASI
-
 import Dispatch
+#endif
+
 import Foundation
 import XCTest
 
 func race(times: Int = 100, _ bodies: () -> Void...) {
+    #if WASI
+    for body in bodies {
+        for _ in 0..<times {
```

Comment:
Not sure if this loop is really doing anything useful if there's no actual race occurring?

Reply:
Otherwise the semantics would be different.